charts/external-dns/templates/deployment.yaml

```diff
@@ -111,6 +111,9 @@ spec:
             {{- if .Values.namespaced }}
             - --namespace={{ .Release.Namespace }}
             {{- end }}
+            {{- if .Values.gatewayNamespace }}
+            - --gateway-namespace={{ .Values.gatewayNamespace }}
```

Comment:
I think one of the issues is, that Gateway actually uses `--gateway-namespace` when it should be unified and `--namespace` is just enough. This is just a confusion.